wallet/src/components/Transactions/index.jsx

```diff
@@ -157,40 +158,63 @@ const Transactions = () => {
     setTxs(mappedTxs);
   }, delay);
 
+  const handleExportIndedexDB = async () => {
+    const exportedDB = await exportIndexdDB('nightfall_commitments');
```

Comment:
Hey @luizoamorim, instead of exporting the data straight away, what do you think of confirming before doing it?

Reply:
Like a modal after the user click in the button? @israelboudoux